main-app/src/main/java/com/nyctransittracker/mainapp/service/StopService.java

```diff
@@ -0,0 +1,26 @@
+package com.nyctransittracker.mainapp.service;
+
+import com.nyctransittracker.mainapp.entity.Stop;
+import com.nyctransittracker.mainapp.repository.StopRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Random;
+
+@Service
+@RequiredArgsConstructor
+public class StopService {
+
+    private final StopRepository repository;
+    public void createStation(Stop stop, String direction, String route) {
+        var newStop = new Stop();
+        newStop.setUniqueId(new Random().nextLong());
```

Comment:
same comment about generated id from Stop.java

Reply:
I realized this was creating more than one instance of a stop so I changed the id to be a string that combines the GTFS stop id, route and direction. This is working!